ServerCore/Pages/PuzzleApiController.cs

```diff
@@ -91,6 +91,7 @@ public async Task<ActionResult<MailInfo>> GetMailInfoAsync(string eventId, int p
             MailInfo mailInfo = new MailInfo();
 
             mailInfo.PuzzleName = puzzle.Name;
+            mailInfo.PuzzleSupportAlias = puzzle.SupportEmailAlias ?? currentEvent.ContactEmail ?? "puzzhunt@microsoft.com";

```

Comment:
Are we switching from all mail going to puzzhunt to sending it to atuhors?

Reply:
It's a per-puzzle choice. We did this last year for puzzles supported by people who couldn't get into the puzzhunt account, and in the Discord you can see a discussion about doing it this year as well. This code mirrors line 18 of Submissions/Index.cshtml, so that the new in-puzzle link will work the same as the website-based link.

Reply:
And also, I see you've got puzzhunt in the support list for your puzzles, so it'll go there anyway because of that.

Reply:
Yes, but that's just our Puzzleday convention. In module-based events, for example, the authors would not put puzzhunt in the list. I know that help.js is PD2022-specific at this point, but this change allows any other event to build a similar experience if they wish.

Reply:
What do we want for puzzleday? To: author; puzzhunt? Just the author seems dangerous

Reply:
The convention we are using is to put puzzhunt as well as the author (if external) as well as anyone who is supporting the author (if external). But that is a policy issue that has nothing to do with the code change...